src/components/Sidebar/Sidebar.tsx

```diff
@@ -112,6 +112,23 @@ export const Sidebar = ({ mentors, handleModal }: SidebarProps) => {
           handleModal={openModal}
         />
       </SidebarNav>
+      <a
+        href="https://vercel.com/?utm_source=[coding-coach]&utm_campaign=oss"
+        aria-label="Vercel"
+        target="blank"
```

Comment:
Today I learned that some browsers will treat `target="blank"` as `target="_blank"`.  I'm not sure how I feel about that yet, but for now I think we should make sure to stick to the supported options in the spec.

More specifically, the change below:

```suggestion
        target="_blank"
```

Reply:
For clarity, we'd only need to use the second suggestion; not the first.